core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.template.html

```diff
@@ -25,17 +25,17 @@ <h3>
       <strong>Enter the url fragment for the subtopic</strong>
     </div>
     <input class="form-control protractor-test-new-subtopic-url-fragment-field" style="font-size: 12px;"
-           ng-class="{'is-invalid': $ctrl.editableUrlFragment && !$ctrl.isUrlFragmentValid()}"
+           ng-class="{'is-invalid': $ctrl.editableUrlFragment && (!$ctrl.isUrlFragmentValid() || $ctrl.subtopicUrlFragmentExists)}"
            placeholder="Enter a url fragment for the subtopic."
            ng-model="$ctrl.editableUrlFragment" ng-change="$ctrl.checkSubtopicExistence()" maxlength="<[$ctrl.MAX_CHARS_IN_SUBTOPIC_URL_FRAGMENT]>">
     <span class="oppia-input-box-subtitle">
       <i>
         The subtopic URL fragment is used to uniquely access the subtopic viewer page. It should consist of one or more hyphen-separated words, all in lowercase, with at most <[$ctrl.MAX_CHARS_IN_SUBTOPIC_URL_FRAGMENT]> characters in total and must be unique across the topic. Please use meaningful keywords, and avoid using words like "and", "of", or "the".
       </i>
     </span>
-    <span class="form-text error-msg" ng-if="$ctrl.editableUrlFragment && $ctrl.subtopicUrlFragmentExists">
-      <em>The subtopic url fragment is not unique.</em>
-    </span>
+    <div class="oppia-input-box-subtitle text-danger" ng-if="$ctrl.editableUrlFragment && $ctrl.subtopicUrlFragmentExists">
+      <i>The subtopic url fragment is not unique.</i>
```

Comment:
Usually we use "em" for semantic emphasis; i is just for styling. Why did you change it?

Please fix for other instances of the "i" tag.

Reply:
In create-new-topic-modal.template and in create-new-topic-modal.template, we use <i> inside <spans> with the class "oppia-input-box-subtitle". I just folllowed the same pattern. Please let me know if this has to be changed to <em> in all of these places too. Thanks!

Reply:
Interesting -- after reading up on "i" vs "em", I'd say use "em" everywhere.